021_MergeTwoSortedLists.py

```diff
@@ -0,0 +1,80 @@
+class ListNode(object):
+    def __init__(self, x):
+        self.val = x
+        self.next = None
+    def listToArray(self):
```

Comment:
nit: When you have a class method, it's good practice to document your function. For this, use """ <explanation> """ docstring. For example, for more info take a look at: https://realpython.com/documenting-python-code/